README.md

```diff
@@ -14,91 +14,75 @@ The client implement for SkyWalking BanyanDB in Java.
 
 ## Create a client
 
-Create a `BanyanDBClient` with host, port and a user-specified group and then establish a connection.
+Create a `BanyanDBClient` with host, port and then use `connect()` to establish a connection.
 
 ```java
 // use `default` group
-client = new BanyanDBClient("127.0.0.1", 17912, "default");
+client = new BanyanDBClient("127.0.0.1", 17912);
 // to send any request, a connection to the server must be estabilished
-client.connect(channel);
+client.connect();
 ```
```

Comment:
Please add these API docs on the top of the main class comment using `code`, which could be helpful for javadoc generation.

Reply:
Added